### PR TITLE
4.15.3 hotfix

### PR DESCRIPTION
**Pull request by fabiocav**
4.15 Hotfix update

### CODE REVIEW

**Review by pragnagopa — APPROVED** [review]

**Review by liliankasem — APPROVED** [review]

File: build/common.props [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <LangVersion>latest</LangVersion>
     <MajorVersion>4</MajorVersion>
     <MinorVersion>15</MinorVersion>
-    <PatchVersion>1</PatchVersion>
+    <PatchVersion>2</PatchVersion>
```
Comment (VpOfEngineering) [inline build/common.props]: We had a windows-only hotfix as `4.15.2`. Should this be `4.15.3`?

https://github.com/Azure/azure-functions-host/releases/tag/v4.15.2

Comment (fabiocav) [reply build/common.props]: Ah, missed that! Yes, having two with the same version would create some confusion. Let me update.

Comment (fabiocav) [reply build/common.props]: Done.

Comment (kshyju) [reply build/common.props]: change PR title /commit message to reflect new version when merging to avoid confusion

**Review by mathewc — APPROVED** [review]

**Pull request merged**